modules/conversion-center/assets/scss/widgets/contact-buttons.scss

```diff
@@ -32,10 +36,23 @@
 
     position: fixed;
     bottom: 20px;
-    min-width: var(--e-contact-buttons-chat-box-width);
     #{$end}: 20px;
+    width: var(--e-contact-buttons-chat-box-width);
     z-index: 99;
 
+    @media (max-width: $screen-mobile-max) {
+        #{$end}: 0;
```

Comment:
What is `$end/$star`t used for?

Reply:
I was thinking the same thing. 
It is not very clear.

Reply:
This is used for rtl support

Reply:
We tend to use logical css properties to handle RTL and LTR

Reply:
@davseve that's what I started using, but then I added this as per Louis' input (see here: https://github.com/elementor/elementor/pull/25859/files/f4dde61c52703535e45586c72f8f6afefe1fc98d#r1582973479 )
Let me know what you think, I can change it back if needed.

Reply:
@mserino @davseve I believe we can use` inset-inline-start` and` inset-inline-end`. Most important it that our CSS is supported by Safari 14 and upwards, which I believe it is. See: https://caniuse.com/?search=inset-inline-start. 

@rami-elementor Can you please correct me if I am wrong here?

Reply:
@hein-obox I have addressed this comment, would you mind checking again?

Reply:
@hein-obox is correct.

Reply:
@ramiy this is `position: fixed` because the component needs to float on the bottom right or bottom left of the page. Flex doesn't allow that. I specifically need the fixed position.